Tasks/Common/VstsAzureHelpers_/VstsAzureHelpers_.psm1

```diff
@@ -11,8 +11,6 @@ if ($global:DebugPreference -eq 'Continue') {
 # Import the loc strings.
 Import-VstsLocStrings -LiteralPath $PSScriptRoot/module.json
 
-[System.Net.WebRequest]::DefaultWebProxy = Get-VstsWebProxy
```

Comment:
Do we know why it was added? Can this break any exiting scenario?

Reply:
It is added back.